### DISCUSSION TITLE
Getting room/point coordinates - problem

### DISCUSSION THREAD

**Discussion by dragop4** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#233] [Q&A] [answered]
How long i must hold "play" button, for coords? Popup does not appear, in debug console don't see config.
I tested on firefox (with incognito, disabled adblock) and chrome. On Linux.

**Comment by amibumping** [comment]
Happened to me too, try to do double click on the play button. It worked for me

**Reply by dragop4** [reply]
Still not working

**Reply by PiotrMachowski** [reply]
@amibumping was it also on Linux? Maybe some browsers doesn't support long press... Half a second should be enough as far as I remember

**Reply by amibumping** [reply]
> @amibumping was it also on Linux? Maybe some browsers doesn't support long press... Half a second should be enough as far as I remember

Yep, it was on Linux with Firefox

**Comment by dragop4** [comment, marked as answer]
I'm stupid. Play button for zones is on right side...

**Reply by PiotrMachowski** [reply]
Well, that explains a lot 😃

**Comment by shred86** [comment]
I'm able to get the pop-up but it's only providing four numbers, example:

```
      [
        13050,
        25774,
        17634,
        29537
      ]
```

Shouldn't there by an x,y pair for each corner? This with the Roborock S7.

**Reply by PiotrMachowski** [reply]
Nope, rectangle is always in the same orientation, so just 2 opposite corners are needed

**Reply by shred86** [reply]
Ah, makes sense. Took a bit to figure out but I think I got it working now! Very cool.
